### ISSUE TITLE
Tab scrolling setting won't show all options

### ISSUE THREAD

**Issue by jerone** [atom/tabs#284]
The "Tab Scrolling" setting from this package won't show all options:

![atom tabs config](https://cloud.githubusercontent.com/assets/55841/13723258/71d354f0-e85f-11e5-88b5-75721c111138.png)

Looks like Setting package doesn't like the `true` & `false` Boolean property from the `configSchema`: https://github.com/atom/tabs/blob/master/package.json#L29-L30

---

_Atom v1.6.0-beta8 in safe-mode
Windows 7
tabs@0.91.1_

**Comment by thomasjo** [comment]
Nice catch! I can verify that this is also the case on OS X, running 1.7.0-dev-430a0e0.

Not sure if we can easily resolve this here for the time being, or if we'll have to patch this in settings-view (or core). Gut feeling is that this needs to be resolved in settings-view.

**Comment by jerone** [comment]
Haven't tried it, but you can try using Strings instead of Booleans...

**Comment by thomasjo** [comment]
> Haven't tried it, but you can try to using Strings, not Booleans as values...

Yeah, that'll work for the UI, but I'm not sure the non-UI code will work then since `"true" == true` will yield `false` in CoffeeScript. It's easy to test though :smiley:

**Comment by jerone** [comment]
Maybe @maxbrunsfeld has more insides of this per https://github.com/atom/tabs/pull/260

**Comment by puranjayjain** [comment]
@jerone after much going through the code I found the bug is related to html and as @thomasjo highlighted it cannot be just solved by coffeescript directly.

The bug:
https://github.com/atom/settings-view/blob/master/lib/settings-panel.coffee#L246

I found it can be fixed partially:
```coffeescript
@option value: option, option.toString()
```

but then the backend logic would not update it back to it's correct value. (Although the booleans don't work right now either)

So I can draw this conclusion/possible solutions:
* Either all the package owners need to be updated that these values don't work and they should change these boolean values to something else.
* Solve this in atom core and settings

Possible solutions:
https://groups.google.com/forum/#!topic/knockoutjs/_-NzcCQreQ8
http://jsfiddle.net/rniemeyer/H4gpe/
http://stackoverflow.com/questions/10510683/knockoutjs-version-2-1-0-bind-boolean-value-to-select-box
